src/Components/ManageTeam/EmployeesTable.tsx

```diff
@@ -0,0 +1,165 @@
+import AddIcon from '@mui/icons-material/Add';
+import DeleteIcon from '@mui/icons-material/Delete';
+import EditIcon from '@mui/icons-material/Edit';
+import {
+  Button,
+  IconButton,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Toolbar,
+  Tooltip,
+  Typography,
+} from '@mui/material';
+import { useState } from 'react';
+
+import { ConfirmDialog } from './ConfirmDialog';
+import { EmployeeForm } from './EmployeeForm';
+import { Notify } from './Notify';
+import { PopUpEmployeeForm } from './PopUpEmployeeForm';
+
+function createData(id: number, login: string, password: string, role: string) {
+  return { id, login, password, role };
+}
+
+const ROWS = [
+  createData(1, 'dawpal12', 'pasikonik', 'waiter'),
+  createData(2, 'julrybo13', 'samolocik', 'manager'),
+  createData(3, 'kamwene16', 'pandysafajne', 'waiter'),
+  createData(4, 'szymkno21', 'kotlet', 'cook'),
+  createData(5, 'ewetryb31', 'ziemniaczki', 'cook'),
+];
+
+type NewEmployeeData = {
```

Comment:
`NewEmployeeData` could also have only one source. This source could be for instance `models/user.d.ts`.